### PR TITLE
Update guidelines.md

### PR DESCRIPTION
**Pull request by apeltzer**
Adding in an acknowledgement section

### CODE REVIEW

**Review by MaxUlysse — APPROVED** [review]

File: markdown/developers/guidelines.md [hunk 1]
```diff
@@ -102,3 +102,20 @@ However, in general, pipelines must:
...
+We won't change the way how nf-core pipelines are built (open, free for all, permissive licence (MIT), ...), but wanted to make sure that authors of original code (whether pipeline specific or tool specific / script specific) receive the credit and acknowledgement they deserve for their work. Always try to remember, that a large portion of the work that is available to anyone via nf-core is in many cases freelance work of enthusiastic developers around the globe. Please act accordingly - you don't "pay" for the work they did, but:
+
+**The least you can do when using their work (or amending it to your own needs) is to credit them and their contributors properly.** 
+
+Some examples and on how you could do that to make sure they feel valued (list not necessarily complete):
+
+* Send them a message via Slack and let them know that you use their work & had to change something to fit your own purpose. 
+* If you use portions of pipeline code, even if its just tiny pieces:
+  * Link to the original repository and/or authors
+  * Leave existing credits and acknowledgement sections intact - usually there are many more than just a single author involved, which deserve credits too!
```
Comment (apeltzer) [inline markdown/developers/guidelines.md:119]: One might add here that people can add themselves of course if they changed / vendored e.g. a pipeline for internal use.

File: markdown/developers/guidelines.md [hunk 2]
```diff
@@ -102,3 +102,20 @@ However, in general, pipelines must:
...
+We won't change the way how nf-core pipelines are built (open, free for all, permissive licence (MIT), ...), but wanted to make sure that authors of original code (whether pipeline specific or tool specific / script specific) receive the credit and acknowledgement they deserve for their work. Always try to remember, that a large portion of the work that is available to anyone via nf-core is in many cases freelance work of enthusiastic developers around the globe. Please act accordingly - you don't "pay" for the work they did, but:
+
+**The least you can do when using their work (or amending it to your own needs) is to credit them and their contributors properly.**
+
+Some examples and on how you could do that to make sure they feel valued (list not necessarily complete):
+
+* Send them a message via Slack and let them know that you use their work & had to change something to fit your own purpose.
+* If you use portions of pipeline code, even if its just tiny pieces:
+  * Link to the original repository and/or authors
+  * Leave existing credits and acknowledgement sections intact - usually there are many more than just a single author involved, which deserve credits too!
```
Comment (apeltzer) [inline markdown/developers/guidelines.md:119]:
```suggestion
  * Leave existing credits and acknowledgement sections intact - usually there are many more than just a single author involved, whom deserve credits too!
```

File: markdown/developers/guidelines.md [hunk 3]
```diff
@@ -102,3 +102,19 @@ However, in general, pipelines must:
   * The nf-core `dev` branch should be set as the default branch up until the first release
 * Use GitHub releases and [keep a detailed changelog](https://keepachangelog.com/en/1.0.0/) file
 * Follow a versioning approach, e.g. [Semantic Versioning](https://semver.org/) for your pipeline releases
+
+## Credits & Acknowledgements
+
+Most pipelines within the nf-core community have adopted a permissive MIT licence that permits reusing any pipeline also without an academic background. However, we want to emphasize that using prior work in nf-core pipelines from other pipelines / projects without acknowledging the original author(s) is considered a bad practice by the community.
```
Comment (jfy133) [inline markdown/developers/guidelines.md:108]:
```suggestion
Most pipelines within the nf-core community have adopted a permissive MIT licence that permits reusing any pipeline also without an academic background. However, we want to emphasize that using or adopting/adapting **previous work** in nf-core pipelines from other pipelines / projects without acknowledging the original author(s) is considered a bad practice by the community.
```

File: markdown/developers/guidelines.md [hunk 4]
```diff
@@ -102,3 +102,19 @@ However, in general, pipelines must:
...
+
+We won't change the way how nf-core pipelines are built (open, free for all, permissive licence (MIT), ...), but wanted to make sure that authors of original code (whether pipeline specific or tool specific / script specific) receive the credit and acknowledgement they deserve for their work. Always try to remember, that a large portion of the work that is available to anyone via nf-core is in many cases freelance work of enthusiastic developers around the globe. Please act accordingly - you don't "pay" for the work they did, but:
```
Comment (jfy133) [inline markdown/developers/guidelines.md:110]:
```suggestion
We won't change the way how nf-core pipelines are built (open, free for all, permissive licence (MIT), ...), but want to make sure that authors of original code (whether pipeline specific or tool specific / script specific) receive the credit and acknowledgement they deserve for their work. Always try to remember, that a large portion of the work that is available to anyone via nf-core is in many cases freelance work of enthusiastic developers around the globe. Please act accordingly - you don't "pay" for the work they did, but:
```

**Review by jfy133 — APPROVED** [review]
Do you also want to mention that you should explictly check what the license the previous work _may_ have? As this may influence how the credit/acknowledgment should be written?

**Comment by apeltzer** [comment]
Ok we can still polish afterwards, but this seems to cover at least some aspects that are seemingly not understood by some people ;-)

**Pull request merged**